rl_algo_impls/microrts/vec_env/microrts_bot_vec_env.py

```diff
@@ -56,6 +56,7 @@ def __init__(
         cycle_maps=[],
         video_frames_per_second: Optional[int] = None,
         non_deterministic: bool = False,
+        render_mode: Optional[str] = None,
```

Comment:
Instead of setting the default value down at line 83, how about making render_mode: str = "rgb_array"? This is what I do with https://github.com/sgoodfriend/rl-algo-impls/blob/cacb1f0ecc88072c4680d2aea1341ae07f01fb65/rl_algo_impls/lux/vec_env/jux_vector_env.py#L54